README.md

```diff
@@ -1,5 +1,7 @@
 # Hacktober Fest 2022 Participants
 
+<img src="https://github.com/12-malak/hacktober-fest-2022-participants/blob/main/Logo.png" alt="hi" class="inline"/>
```

Comment:
The image should be in the root foolder of this repo. Not a link from a different repository. 

Also, kindly post the logo in the PR as well thanks.